---
layout: default
title: Programme
subtitle: Programme for Genome Science 2019
use-site-title: true
---

### Tuesday 3rd September 2019

__Start__|__End__|__Who__|__Title__
-------|-------|-------|-------
**_11:30_**|**_12:20_**|**_Registration and lunch_**|
12:20|12:30|Mick Watson|Introduction and Welcome
12:30|13:20|**Keynote:** Wendy Bickmore|Understanding function in the non-coding genome
13:20|14:10|**Keynote:** Sarah Teichmann|Immunogenomics one cell at a time
*14:10*|*14:40*|*Coffee*| 
**_14:40_**|**_16:10_**|**_Evolving Technologies 1_**| 
14:40|15:00|Clive Brown (Oxford Nanopore)|The latest from Nanopore sequencing
15:00|15:20|Vince Smith (Illumina)|Advances in sequencing technology that are transforming genomics
15:20|15:35|Louise Williams (New England BioLabs)|New England BioLabs
15:35|15:45|BioNano|BioNano
15:45|16:10|Omer Bayraktar|The Rise of Spatial Genomics
*16:10*|*16:40*|*Coffee*| 
**_16:40_**|**_18:00_**|**_Evolving Technologies 2_**| 
16:40|17:00|Rade Drmanac (BGI/Complete):|A new generation of NGS: PCR free DNBseq + CoolNGS chemistry with unlabeled nucleotides + Single-tube unique cobarcoding (stLFR).
17:00|17:15|Samplix|Samplix
17:15|17:35|Tuval Ben Yehezkel (LoopSeq):|LoopSeq Synthetic Long Read Sequencing and Its Applications: from Microbiome to Transcriptome and Beyond
17:35|17:45|TTP LabTech|TTP LabTech
17:45|18:00|Mike Quail|PacBio Sequel II, 8M and tree of life
**_18:00_**|**_19:30_**|**_Wine, snacks, and posters_**|

---

### Wednesday 4th September 2019

#### Parallel sessions

__Time__|__A__|__B__
-----|-----|-----
09:00 - 10:40|*Plant and Animal Genomics*|*Evolutionary Genomics*
11:15 - 13:15|*Clinical Genomics*|*Microbes I*
14:15 - 16:00|*Developmental Biology*|*Microbes II*
16:20 - 18:00|*Bioinformatics and software* |*Genome Engineering*

---

#### Session A

**Start**|**End**|**Who**|**Title**
-------|-------|-------|-------
**_09:00_**|**_10:40_**|**_Plant and animal genomics_**| 
09:00|09:20|Emily Humble|horned oryx conservation genomics (TBC)
09:20|09:40|Rachel Gilroy|The broiler chicken intestinal microbiome: interventions to improve health and welfare
09:40|09:55|Jon Rock (Lexogen)|Improving the quality of your RNA Next Generation Sequencing with Lexogen
09:55|10:15|Matthew Parker|Increasing recombination in the barley (Hordeum vulgare) by manipulating meiotic genes
10:15|10:35|Miriam Schreiber|Direct RNA Sequencing of the Arabidopsis transcriptome with Nanopores
*10:40*|*11:15*|*Coffee*|
**_11:15_**|**_13:15_**|**_Clinical Genomics_**| 
11:15|11:35|Sian Ellard|Application of genomic sequencing technology to the diagnosis of rare diseases: from the 100,000 Genomes Project to a nationally commissioned Genomic Medicine Service
11:35|11:50|Ben Harvey (Agilent)|Agilent’s workflow solutions for the clinical genomics laboratory
11:50|12:10|Mike Inouye|The utility of polygenic risk scores for cardiovascular disease
12:10|12:25|Fluidigm|Fluidigm
12:25|12:45|Sergey Koren|Telomere-to-telomere assembly of complete human chromosomes
12:45|13:05|Giordano Bottà|Integration of Polygenic Risk Score in CAD risk models for clinical use
13:05|13:15|Stilla|Stilla
*13:15*|*14:15*|*Lunch*|
**_14:15_**|**_16:00_**|**_Developmental Biology_**| 
14:15|14:40|Alistair McGregor|Investigating the developmental consequences of whole genome duplication in arachnids
14:40|15:05|Mansi Srivastava|Acoel genome reveals the regulatory landscape for whole-body regeneration
15:05|15:15|Qiagen|Qiagen
15:15|15:35|Tamir Chandra|Mouse embryonic stem cells switch from naïve to formative state during transition through G2M
15:35|15:55|Laura Mincarelli|Combined short and long read single-cell sequencing identify aging related transcriptional profile and splicing landscape in hematopoietic stem cells and progenitors
*15:55*|*16:20*|*Coffee*|
**_16:20_**|**_18:00_**|**_Bioinformatics and Software_**| 
16:20|16:45|Rachel Colquhoun|Nucleotide level analysis of genetic variation in the bacterial pan-genome with Pandora
16:45|17:10|Anton Koroneynikov|Tools for assembly graph analysis and more
17:10|17:35|Apurva Narechania|What do we gain when tolerating loss? The information bottleneck, lossy compression, and detecting horizontal gene transfer
17:35|18:00|Wenbin Guo|3D RNA-seq - a powerful and flexible tool for rapid and accurate differential expression and alternative splicing analysis of RNA-seq data for biologists

---

#### Session B

**Start**|**End**|**Who**|**Title**
-----|-----|-----|-----
**_09:00_**|**_10:40_**|**_Evolutionary genomics_**| 
09:00|09:25|Reuben Nowell|Genome evolution in the bdelloid rotifers: ancient asexuality and genome structure
09:25|09:50|Pramod Shinde|Impact of mitochondrial epistatic interactions on evolution of different human subpopulations
09:50|10:15|Max Stammnitz|Tracing the evolution of two transmissible cancers in Tasmanian devils
10:15|10:40|Fiona Jane Whelan|The coincident (co-occurrence and avoidance) relationships of genes in prokaryote genomes
*10:40*|*11:15*|*Coffee*|
**_11:15_**|**_13:15_**|**_Microbes I_**| 
11:15|11:45|Anna V. Protasio:|Parasites within parasites: transposable elements in platyhelminthes
11:45|12:15|Julie Segre:|Human skin microbiome: trans-kingdom, host-immune interactions
12:15|12:40|Emma Ainsworth|Long-Read Sequencing Determines Genome Structure in Salmonella Typhi
12:40|13:05|Leigh Monahan|Morphoseq enables closed circle bacterial genome assemblies from short read platforms
13:05|13:15|Ralph Vogelsang (PacBio)|PacBio HiFi Long Reads for Metagenomic Discoveries
*13:15*|*14:15*|*Lunch*|
**_14:15_**|**_16:00_**|**_Microbes II_**| 
14:15|14:40|David Gally|Prophage-mediated chromosomal rearrangements in Escherichia coli O157
14:40|15:05|Miren Iturriza-Gomara|Rotavirus vaccines and strain diversity
15:05|15:30|Max Stammnitiz|Scrutinising nanopore sequencing for freshwater microbiomes
15:30|15:55|Tania Duarte|Deep Sequencing of Microbial Communities in Cystic Fibrosis Airways
*15:55*|*16:20*|*Coffee*|
**_16:20_**|**_18:00_**|**_Genome Engineering_**| 
16:20|16:45|Maria Paz Zafra Martin|Engineering the cancer genome at single base resolution
16:45|17:10|Julian Gruenewald |CRISPR DNA base editors with reduced rna off-target effects
17:10|17:35|Mike McGrew|Genome editing in chickens
17:35|18:00|Spring Tan|Genome Wide CRISPR Knockout Screen Identifies Host Factors Involved in Bovine Herpes Virus Type 1 Infection

---

#### Conference Dinner on the evening of Weds 4th - **_ticketed_**

---

### Thursday 5th September 2019

**Start**|**End**|**Who**|**Title**
-----|-----|-----|-----
**_09:00_**|**_11:00_**|**_Single Cell Genomics_**| 
09:00|09:20|Iain Macaulay|Molecular biology, including long reads for scRNA-seq
09:20|09:40|Martin Hemberg|Bioinformatics for scRNA-seq
09:40|09:55|10X Genomics| 
09:55|10:10|Daniel Liber (Takara)|What are you missing from your single cell RNA-seq? Go beyond gene expression with the ICELL8 Single-Cell System!
10:10|10:20|Zhouchun Shang (MGI/BGI)|Dissecting cell heterogeneity using single-cell omics powered by DNBSEQ
10:20|10:40|Kristina Kirschner|Single cell Sequencing reveals Notch mediated secondary senescence
10:40|11:00|Daniel Seaton|Mapping the functions of human disease risk alleles during neuronal development using multiplexed differentiation of human induced pluripotent stem cells
*11:00*|*11:30*|*Coffee*| 
11:30|12:20|**Keynote:** Jane Carlton|Bringing genomics into malaria field studies in India
12:20|13:10|**Keynote:** Kirsten Bos|Genomic reconstructions of ancient pathogens
**_13:10_**|**_THE END_**| |**_Packed lunch available_**
